LAPACKE/src/lapacke_cgeqrt_work.c

```diff
@@ -76,14 +76,15 @@ lapack_int LAPACKE_cgeqrt_work( int matrix_layout, lapack_int m, lapack_int n,
         }
         /* Transpose input matrices */
         LAPACKE_cge_trans( matrix_layout, m, n, a, lda, a_t, lda_t );
+        LAPACKE_cge_trans( matrix_layout, nb, MIN(m,n), t, ldt, t_t, ldt_t );
```

Comment:
`t_t` is output only, right? If so, there is no need for this line. The same happens in the other 3 files.

Reply:
@sbite0138, if you agree with me, I can make the change. Thanks